docs/usage/getting-started.md

```diff
@@ -211,7 +236,8 @@ We are going to use Grafana to query the data we wrote into Observatorium. To st
 docker run -p 3000:3000 grafana/grafana:7.3.7
 ```
 
-- Now open your web browser and go to `http://localhost:3000`. the default username:password is `admin:admin`.
-- Add a new Prometheus data source with url `http://172.17.0.1:8080/api/metrics/v1/test-oidc`. We are using `172.17.0.1` as the host because we are trying to access the `token-refresher` running on the host from Grafana, which is running inside a docker container.
+* Now open your web browser and go to `http://localhost:3000`. the default username:password is `admin:admin`.
+* Add a new Prometheus data source with url `http://172.17.0.1:8080/api/metrics/v1/test-oidc`. 
+  * We are using `172.17.0.1`(linux) or `host.docker.internal`(mac) as the host because we are trying to access the `token-refresher`; running on the host, from Grafana; which is running inside a docker container.
```

Comment:
We are using semi-colons to separate dependent clauses, which is awkward, as clauses separated by semi-colons should always be independent, i.e. have a full subject-verb-object structure.

Reply:
We are using capital `D` Docker elsewhere, so let's be consistent:
```suggestion
  * We are using `172.17.0.1`(linux) or `host.docker.internal`(mac) as the host because want to to access the `token-refresher`, which is running on the host, from Grafana, which is running inside a Docker container.
```